src/Integration.UnitTests/MefServices/VsSessionHostTests.cs

```diff
@@ -255,7 +255,7 @@ public void VsSessionHost_ResetBinding_NoOpenSolutionScenario()
             this.stepRunner.AbortAllNumberOfCalls.Should().Be(0);
 
             // Act
-            tracker.SimulateActiveSolutionChanged(isSolutionOpen: false);
```

Comment:
Just a note: VsSessionHost has been deleted on the `hardening-8-6`.

Reply:
Oh it has? That's great. Maybe I should rebase.